### DISCUSSION TITLE
Form submission with button component.

### DISCUSSION THREAD

**Discussion by Pieter3033** [sqlpage/SQLPage#112] [Q&A] [answered]
Hi, 

Is it possible to do a form submission with the button component, I know there is already buttons implemented inside of the form but I would like to do a form submission with the fancy new buttons.

Regards

**Comment by lovasoa** [comment, marked as answer]
Hello ! This is not yet implemented, but I created e new feature request issue for this idea https://github.com/lovasoa/SQLpage/issues/113
